frontend/src/components/Layout.tsx

```diff
@@ -1,18 +1,50 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { styled } from '@mui/material/styles';
 import { useLocation } from 'react-router-dom';
 import ScopedCssBaseline from '@mui/material/ScopedCssBaseline';
 import { Header, GovBanner } from 'components';
+import { useUserActivityTimeout } from 'hooks/useUserActivityTimeout';
+import { useAuthContext } from 'context/AuthContext';
+import UserInactiveModal from './UserInactivityModal/UserInactivityModal';
 
 interface LayoutProps {
   children: React.ReactNode;
 }
+
 export const Layout: React.FC<LayoutProps> = ({ children }) => {
+  const { logout, user } = useAuthContext();
+  const [loggedIn, setLoggedIn] = useState<boolean>(
+    user !== null && user !== undefined ? true : false
+  );
+  const { isTimedOut, resetTimeout } = useUserActivityTimeout(
+    14 * 60 * 1000, // set to 14 minutes of inactivity to notify user
```

Comment:
Should we refactor this out into a separate file `constants.ts` so it's easy to change in the future?